R/parents_of_children.R

```diff
@@ -0,0 +1,74 @@
+require(data.table)
+
+#' @name parents_of_children
+#' 
+#' @description Vectorized implementation of "parents_of_child"
+#' 
+#' @param child_location_id [int] Location ID to pull parent ID for
```

Comment:
can we match the argument name below, and clarify that a vector of location_ids can be used?

Reply:
Good catch; thanks.